src/org/usfirst/frc4904/sovereignty/FusedSensor.java

```diff
@@ -0,0 +1,50 @@
+package org.usfirst.frc4904.sovereignty;
+
+
+public class FusedSensor<T> implements Fusible<T> {
```

Comment:
While I understand the theoretical motivations for using a generic here, are there any cases we have encountered that use anything except a double?